src/components/cylc/tree/cylc-tree.js

```diff
@@ -338,16 +340,17 @@ class CylcTree {
     // NOTE: when deleting the root family, we can also remove the entire cycle point
     if (familyProxyId.endsWith('|root')) {
       // 0 has the owner, 1 has the workflow Id, 2 has the cycle point, and 3 the family name
-      const [owner, workflowId, cyclePoint] = familyProxyId.split('|')
```

Comment:
Removed `cyclePoint` as I don't need that, since I'll build the full cycle point ID